src/workloads/scale/LargeScaleLongLived.yml

```diff
@@ -0,0 +1,82 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 400
+
+Actors:
+- Name: LongLivedCreator
+  Type: Loader
+  Threads: 10
+  Phases:
+  - Repeat: 1
+    Database: &DB longlived
+    CollectionCount: &CollectionCount 1000
+    Threads: 10
+    DocumentCount: &DocumentCount 1000
+    BatchSize: 1000
+    Document:
+      # Each document ranges in size from about 90 to 150 bytes (average 120)
+      x0: &rand_10k_int {^RandomInt: {min: 0, max: 10000}}
+      x1: &rand4int {^RandomInt: {min: 0, max: 2147483647}}
```

Comment:
Why the string rand4int? It's a 32bit integer, yes? Is the 4 for 4 bytes?
It really doesn't matter, just confused me for a moment.

Reply:
I'm renaming this to rand_4b_int to be clearer.  Yes, 4 for 4 bytes.